docs/Homebrew-Governance.md

```diff
@@ -78,20 +82,28 @@
 
 ## 6. Project Leader
 
-1. The Project Leader will represent Homebrew publicly, and manage all day-to-day technical decisions related to the operation of Homebrew.
+1. The Project Leader will represent Homebrew publicly, manage all day-to-day technical decisions, and resolve disputes related to the operation of Homebrew between maintainers, members, other contributors, and users.
 
 2. The Project Leader will be elected annually by a majority vote of Homebrew members. The PLC will nominate at least one candidate for Project Leader. Any member may nominate a candidate, or self-nominate. Nominations must be announced to the membership three weeks before the AGM.
 
 3. Any vacancy of the Project Leader will be filled by appointment of the PLC.
 
-4. After March 1, 2020, the Project Leader may not be a member of the PLC.
+4. The Project Leader may not be a member of the PLC.
+
+5. A technical decision of the Project Leader may be overruled by an ordinary resolution of the TSC.
+
+6. A non-technical decision of the Project Leader may be overruled by an ordinary resolution of the PLC.
```

Comment:
Should there be a definition of what a technical decision/matter/question/dispute is? For instance, what if there were a dispute over whether an issue is technical?

Reply:
> For instance, what if there were a dispute over whether an issue is technical?

That would be a highly specific case, in that it would need to be a decision made by the Project Leader that either the PLC or TSC would want to overrule but the other committee wouldn’t want to (because if both agree, the distinction wouldn’t matter).

That seems unlikely to me, and that if it ever happened we could revert to a vote by all maintainers on if the dispute is of a technical nature or not, at which point we’d be able to use that experience to build a proper definition which fit Homebrew.

Reply:
>That would be a highly specific case

This is true for items 6.5 and 6.6. Note, however, that TSC jurisdiction over technical issues is relevant not only for decisions made by the Project Leader but also for those between members. See 7.1 and 7.3 below.